### DISCUSSION TITLE
[prosemirror-model]: NodeType is missing its groups property

### DISCUSSION THREAD

**Discussion by nathggns** [DefinitelyTyped/DefinitelyTyped#58892] [Issues with a @types package]
There is a groups property on instances of NodeType which include the parsed content expression for a node type's group. This is missing from the types.

[a comment by an automated account is not shown]

**Comment by Narretz** [comment]
Do you want to add it? Or is it a complicated type?

**Comment by ocavue** [comment]
@nathggns `NodeType.groups` is a private filed. This field is not shown in the [document](https://prosemirror.net/docs/ref/#model.NodeSpec) so we won't add it to DefinitelyTyped too. 

You can see that all public fields are commented with `::` in the source code of prosemirror

https://github.com/ProseMirror/prosemirror-model/blob/b9085fbf195e07f336c48647a4b318c8d6133054/src/schema.js#L55-L60 

```
    // :: NodeSpec
    // The spec that this type is based on
    this.spec = spec

    this.groups = spec.group ? spec.group.split(" ") : []
```
